test/e2e/csv_e2e_test.go

```diff
@@ -90,9 +93,11 @@ var _ = Describe("ClusterServiceVersion", func() {
 				return ctx.Ctx().Client().Create(context.Background(), &og)
 			}).Should(Succeed())
 
+			apiname := genName("api")
```

Comment:
Does this need to be `apiname = genName("api")` as we declared apiname as a top-level variable for this when container?

Reply:
Yes, I'll move the apiname to top and remove ':'.